modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -903,7 +904,11 @@ public void testState() throws Exception {
 
         assertEquals(EXIT_CODE_OK, execute("--state"));
 
-        assertContains(log, testOut.toString(), "Cluster is inactive");
+        Pattern patternInactive = Pattern.compile("Cluster state: INACTIVE");
```

Comment:
Can we replace it with smth like

```
        Function<String, Pattern> patternBuilder = (state) -> Pattern.compile("Cluster state: " + state + "\\s+");
        assertTrue(patternBuilder.apply("INACTIVE").matcher(testOut.toString()).find());

```

Will it work on Windows too?

Reply:
Yes, it will. Both tests have been simplified by means of ```Function```.